src/FileConverter.ts

```diff
@@ -200,7 +200,10 @@ export class FileConverter {
         const fileOutputPath: string = output;
         const errorData: string[] = [];
         const converted: boolean = await new Promise<boolean>((resolve) => {
-            const proc: ChildProcess = spawn(multiToolPath, ["transform", `"${doc.uri.fsPath}"`, "-o", `"${fileOutputPath}"`, "-p", "-f"]);
+            // If you are tempted to put quotes around these strings, please don't as "spawn" does that internally.
+            // Something to consider is adding an option to the SARIF viewr so the path to the multi-tool
+            // can be over-ridden for testing.
```

Comment:
Should be able to intercept and mock `imports` during testing. At least it's possible in Jest.

Reply:
Right, but the goal is to be able to test the "real Marketplace extension" with work in the SDK in progress. I'll open an issue with comments. Certainly not doing it in this PR.

---
In reply to: [406538342](https://github.com/microsoft/sarif-vscode-extension/pull/228#discussion_r406538342) [](ancestors = 406538342)

Reply:
Understood. That was definitely not my immediate interpretation of the comment phrasing.